src/brouwers/kitreviews/models.py

```diff
@@ -21,28 +24,28 @@ class KitReview(models.Model):
     raw_text = models.TextField(
         _(u'review'),
         help_text=_('This is your review. You can use BBCode here.')
-        )
+    )
     html = models.TextField(blank=True, help_text=u'raw_text with BBCode rendered as html')
-    positive_points = models.TextField(_('positive points'), blank=True)
-    negative_points = models.TextField(_('negative points'), blank=True)
-    rating = models.PositiveSmallIntegerField(_(u'rating'), default=DEFAULT_RATING)
+    prop = models.ManyToManyField('KitReviewProperty', blank=True, related_name='reviews')
+    rating = models.PositiveSmallIntegerField(_(u'rating'), default=DEFAULT_RATING,
```

Comment:
Also applies to the m2m with `KitReviewProperty`, but an extra model will be required that serves as `through` model.

This model should hold the fk to `KitReview` and `KitReviewProperty`, and the rating should be stored as well. This effectively stores more information about the m2m relation. I may have overseen that myself in the specifications, if that's the case, I'm sorry :-)

I'm thinking that the 'general' rating should be determined based on the mean rating from the kit properties that are relevant to the rating, so maybe `KitReviewProperty` needs an extra field that determines if it's a rating property or not. Example: fitting is a rating property, while 'box difficulty' is not.